internal/functions/serve/serve.go

```diff
@@ -297,21 +311,39 @@ func ServeFunctions(ctx context.Context, envFilePath string, noVerifyJWT *bool,
 	if importMapPath != "" {
 		binds = append(binds, filepath.Join(cwd, importMapPath)+":"+dockerImportMapPath+":ro,z")
 	}
+
+	var mainFuncImportMapPath = ""
+	if importMapPath != "" {
+		mainFuncImportMapPath = dockerImportMapPath
+	}
+	var mainFuncBuf bytes.Buffer
+	if err := mainFuncTemplate.Execute(&mainFuncBuf, mainFuncVars{
+		FuncDir:       relayFuncDir,
+		ImportMapPath: mainFuncImportMapPath,
+	}); err != nil {
+		return err
+	}
+
 	// 4. Start container
 	fmt.Println("Serving " + utils.Bold(utils.FunctionsDir))
-	cmd := []string{"start", "--dir", relayFuncDir, "-p", "8081"}
+	cmd := "edge-runtime start --main-service /home/deno/main -p 8081"
 	if importMapPath != "" {
-		cmd = append(cmd, "--import-map", dockerImportMapPath)
+		cmd = cmd + " --import-map " + dockerImportMapPath
 	}
 	if viper.GetBool("DEBUG") {
-		cmd = append(cmd, "--verbose")
+		cmd = cmd + " --verbose"
```

Comment:
this concats feel hackish. Feel free to suggest a cleaner way to do it.

Reply:
I would prefer keeping cmd as an array of strings then have another variable `cmdString = strings.Join(cmd, " ")`